src/limitedmap.h

```diff
@@ -66,8 +66,7 @@ class limitedmap
     }
     void update(const_iterator itIn, const mapped_type& v)
     {
-        // TODO: When we switch to C++11, use map.erase(itIn, itIn) to get the non-const iterator.
-        iterator itTarget = map.find(itIn->first);
+        iterator itTarget = map.erase(itIn, itIn);
```

Comment:
It is totally confusing though,  but I assumed for that comment to exist,  this had already been bikeshed as somehow sensible.